kratos/tests/geometries/test_triangle_3d_3.cpp

```diff
@@ -360,89 +360,89 @@ namespace Testing
     */
     KRATOS_TEST_CASE_IN_SUITE(Triangle3D3ParallelNoIntersection, KratosCoreGeometriesFastSuite) {
         Triangle3D3<Point > triangle_1(
-            GeneratePoint<NodeType >(0.0, 0.0, 0.0),
-            GeneratePoint<NodeType >(10., 0.0, 2.0),
-            GeneratePoint<NodeType >(0.0, 1.0, 0.0)
+            Kratos::make_shared<Point>(0.0, 0.0, 0.0),
```

Comment:
btw, note that here i use make_shared since Point is NOT derived from intrusive_base